lib/iwdp.js

```diff
@@ -0,0 +1,81 @@
+import { exec, SubProcess } from 'teen_process';
+import EventEmitter from 'events';
+import logger from './logger';
+import request from 'request-promise';
+import { retryInterval } from 'asyncbox';
+
+const IWDP_CMD = 'ios_webkit_debug_proxy';
+const MAX_RETRIES = 10;
+
+class IWDP extends EventEmitter {
+
+  constructor (udid = null) {
+    super();
+    this.udid = udid;
+    this.attempts = 0;
+    this.port = 27753;
```

Comment:
let's pull out default constants like this to constant vars at the top of the file (if we still want this defined in this file at all to begin with)